setup.py

```diff
@@ -74,9 +74,22 @@ def run(self):
         install.run(self)
 
 
+class LlvmliteClean(Command):
+    """Custom clean command to tidy up the project root."""
+    user_options = []
+    def initialize_options(self):
+        pass
+    def finalize_options(self):
+        pass
+    def run(self):
+        os.system('rm -vrf ./build ./dist ./ffi/*.so ./*.egg-info')
```

Comment:
The setuptools clean seemd to miss the library. About windows, yeah, didn't had that in mind. If the patch is welcome in general, I'd fix that.